.azure-pipelines/release-cli.yaml

```diff
@@ -35,7 +35,20 @@ variables:
       value: 'latest'
     ${{ else }}: # Version from the tag. Can be a branch name. Only mainline branches use latest
       value: '$(Build.SourceBranchName)'
-  - group: ESRP
+
+parameters:
+  - name: forceSignOutput
+    displayName: Force Signing Build Output
+    type: boolean
+    default: false
+  - name: notarizeAfterSign
+    displayName: Notarize Build Output
+    type: boolean
+    default: true
+  - name: simulate
+    displayName: Simulate operations (faster)
+    type: boolean
+    default: false
```

Comment:
can we maybe use `skipLongRunningOperations`, at least in the display name? The context you have given in the PR description is not easy to deduce from the `name` and `displayName` here.

Reply:
But it doesn't just skip operations. The most important thing it does is to mock building and signing steps.

Reply:
Maybe we can call it dryRun?